dp/drivers/i40e.c

```diff
@@ -34,6 +34,12 @@
 #include <rte_config.h>
 #include <rte_ethdev.h>
 
+/* DPDK i40e includes */
+#define I40E_MAC_X722_VF 8 //FIXME
```

Comment:
Maybe you are not `#include`ing something? Copy-pasting a `#define` should be our last resort. Maybe you need to say `#define X722_SUPPORT` (move it from the gcc command line here)? But is it necessary?